File: 1470-shuffle-the-array/1470-shuffle-the-array.java
class Solution {
    public int[] shuffle(int[] nums, int n) {
        int[] arr1= new int[nums.length];
        int k=0,i=0,j=1;
        while(k<n){
            arr1[i]=nums[k];
            arr1[j]=nums[k+n];
            i+=2;
            j+=2;
            k++;
        }
        return arr1;
    }
}

File: 1470-shuffle-the-array/1470-shuffle-the-array.js
/**
 * @param {number[]} nums
 * @param {number} n
 * @return {number[]}
 */
var shuffle = function(nums, n) {
  let nums1=[];
  let l1=nums.splice(0,n);
  console.log(l1);
  let l2=nums.splice(-n);
  console.log(l2);
  for(let i=0;i<n;i++){
    nums1.push(l1[i]);
    nums1.push(l2[i]);
  }
  return nums1;
};